SingularityUI/app/components/common/table/UITable.jsx

```diff
@@ -0,0 +1,325 @@
+import React, { Component, PropTypes } from 'react';
+import Waypoint from 'react-waypoint';
+import classNames from 'classnames';
+import _ from 'underscore';
```

Comment:
Eventually I'd like to replace underscore with lodash as well. Perhaps at the same time.